lib/manager/components/GeneralSettings.js

```diff
@@ -40,9 +40,11 @@ type State = {
 export default class GeneralSettings extends Component<Props, State> {
   state = {}
 
-  _onChange = ({target}: SyntheticInputEvent<HTMLInputElement>) => {
+  _onChange = ({target}: SyntheticInputEvent<HTMLInputElement>, key: 'name' | 'url') => {
+    let value
     // Change empty string to null to avoid setting URL to empty string value.
-    const value = target.value === '' ? null : target.value.trim()
+    if (key === 'url') value = target.value === '' ? null : target.value.trim()
+    else value = target.value
```

Comment:
Would it not be cleaner to always set `value = target.value` unless it's a url in which case set the trim inside an if block? That way it's one fewer check

Reply:
Much cleaner. See https://github.com/ibi-group/datatools-ui/pull/841/commits/07ed8e52df2b04bd228f7bf2d850f03a41005f9a